### PR TITLE
docs: Add setup step 3 to the toolbar

### PR DESCRIPTION
**Pull request by ryan953**
## DESCRIBE YOUR PR
*Tell us what you're changing and why. If your PR **resolves an issue**, please link it so it closes automatically.*

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

Here's a snippet of what the changed pages look like now:
| Page | Before | After |
| --- | --- | --- |
| [index](https://docs.sentry.io/product/dev-toolbar/) | SCR-20241218-jgkh | SCR-20241218-jfiu |
| [setup](https://docs.sentry.io/product/dev-toolbar/setup/) | SCR-20241218-jgrt | SCR-20241218-jfjw

### CODE REVIEW

File: docs/product/dev-toolbar/setup.mdx [hunk 1]
```diff
@@ -36,6 +36,10 @@ You need to complete two steps to get the toolbar rendered on the page:
     </html>
     ```
 
+3. Edit Project Settings in Sentry so the Toolbar is allowed to connect from your dev, staging, and production domains.
```
Comment (JoshFerge) [inline docs/product/dev-toolbar/setup.mdx:39]: is it possible to link to the settings page from here?

Comment (ryan953) [reply docs/product/dev-toolbar/setup.mdx:39]: I can get to `https://sentry.sentry.io/settings/projects/` but without knowing the project name i can't get closer

Comment (JoshFerge) [reply docs/product/dev-toolbar/setup.mdx]: ahh its project settings, not org settings. gotcha. could make it org settings?

Comment (ryan953) [reply docs/product/dev-toolbar/setup.mdx]: it's a per-project config

Comment (ryan953) [reply docs/product/dev-toolbar/setup.mdx]: orgs can have like, their main website, but also all the backend tools with very different domains. kinda a security thing

if different projects are on the same domain that's ok. the request is for a project+domain pair. So everything could have `localhost` added to each of their lists no problem.

Comment (ryan953) [reply docs/product/dev-toolbar/setup.mdx]: the security position is that: 
we're doing a login using sentry.io, so cookies and things never leave our own control. but org data is leaving sentry.io so I wanted to validate that we're exfiltrating those api responses to a domain known to the customer. This way org data only gets sent to domains the customer controls... then the sdk can use the data to render the toolbar.

File: docs/product/dev-toolbar/setup.mdx [hunk 2]
```diff
@@ -36,6 +36,10 @@ You need to complete two steps to get the toolbar rendered on the page:
     </html>
     ```
 
+3. Edit [Project Settings](https://sentry.io/orgredirect/organizations/:orgslug/settings/projects/) in Sentry so the Toolbar is allowed to connect from your dev, staging, and production domains.
```
Comment (billyvg) [inline docs/product/dev-toolbar/setup.mdx:39]: This just redirects me to a random org (I thought it used to ask you to select an org?)

**Review by billyvg — APPROVED** [review]

File: docs/product/dev-toolbar/setup.mdx (hunk 2 above)
Comment (ryan953) [reply docs/product/dev-toolbar/setup.mdx:39]: it should take you to the last org you used, something like that

**Pull request merged**